tools/integration_tests/read_cache/helpers_test.go

```diff
@@ -0,0 +1,86 @@
+package read_cache
+
+import (
+	"path"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/client"
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/log_parser/json_parser/read_logs"
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/operations"
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/setup"
+)
+
+// Expected is a helper struct that stores list of attributes to be validated from logs.
+type Expected struct {
+	StartTimeStampSeconds int64
+	EndTimeStampSeconds   int64
+	BucketName            string
+	ObjectName            string
+	content               string
+}
+
+func readFileAndGetExpectedOutcome(testDirPath, fileName string, t *testing.T) *Expected {
+	expected := &Expected{
+		StartTimeStampSeconds: time.Now().Unix(),
+		BucketName:            setup.TestBucket(),
+		ObjectName:            path.Join(testDirName, fileName),
+	}
+
+	content, err := operations.ReadFileSequentially(path.Join(testDirPath, fileName), MiB)
```

Comment:
For Readability - I would add a variable named chunkSizeToRead and then set it as MiB and pass the variable.